docs/GraFx-Studio/guides/anchoring/index.md

```diff
@@ -0,0 +1,179 @@
+
+## Take your time to explore
+
+Anchoring can be explored by creating a blank document with at least two frames and following along with the guide below.
+
+Select a frame to access anchoring properties in the properties panel.
+
+## The Default
+
+When a new frame (text, image, etc.) is created, anchoring is set to Relative by default.
+
+The default Target is set to Page.
+
+![screenshot-full](anchor_guide_02.png)
+
+## Properties Menu
+
+Select a frame, and you'll see the **Anchoring** section in the properties panel. Additional options may appear based on your selections.
+
+![screenshot-full](anchor_guide_01.png)
+
+## Types of Anchors
+
+Select a frame, then adjust the Horizontal and/or Vertical anchor.
+
+![screenshot-full](anchor_guide_05.gif)
+
+### Relative (%)
+
+The image frame below is anchored to the page by default, with Left and Right set to 5%, Top at 14%, and Bottom at 38%.
+
+![screenshot-full](anchor_guide_03.png)
+
+If the canvas size changes (e.g., from 1080 px to 1200 px), the frame's relative position remains the same at 5% from the left and right. However, the actual pixel distance will change:
+
+- 5% of 1080 = 54 px
+- 5% of 1200 = 60 px
+
+!!! Info "Precision"
+    The anchors on screen only show full percentages. Behind the screens GraFx Studio calculates with **4** decimals precision.
+    
+    54.1432% will display as **54%**
+
+    54.9876% will display as **55%**    
+    
+    You can still enter the exact pixel location of a frame to start, and the percentages will show on the selected frame. (when set to relative)
+    
+
+![screenshot-full](anchor_guide_04.gif)
+
+### Left, Right, Top, Bottom
+
+Selecting these anchors sets the frame to a fixed offset from the chosen side(s).
+
+#### Left
+
+A fixed offset (in pixels, mm, inches, etc.) is set from the left target. If the canvas width changes, the left offset remains constant.
+
+![screenshot-full](anchor_guide_06.gif)
+
+#### Right
+
+A fixed offset is set from the right target. If the canvas width changes, the right offset remains constant.
+
+![screenshot-full](anchor_guide_07.gif)
+
+#### Left & Right
+
+A fixed offset is maintained on both the left and right sides. If the canvas width changes, both offsets remain unchanged.
+
+![screenshot-full](anchor_guide_08.gif)
+
+#### Top
+
+**Note:** Top, Bottom and Top & Bottom are only available.
+
+A fixed offset (in pixels, mm, inches, etc.) is set from the top target. If the canvas height changes, the top offset remains constant.
+
+#### Bottom
+
+A fixed offset is set from the bottom target. If the canvas height changes, the bottom offset remains constant.
+
+#### Top & Bottom
+
+A fixed offset is maintained on both the top and bottom sides. If the canvas height changes, both offsets remain unchanged.
+
+### Center
+
+The selected frame stays centered **relative** to the canvas, and its size does not change if the canvas dimensions change.
```

Comment:
It's relative to the target, which can be the page or a frame. You cannot anchor to the canvas, which is your engine's artboard.

Also, the way it's explained here sounds like it's centered on the target, but you can also place it at f.i. 30%. The center of your frame will stay at 30% in this case.

It's indeed correct and good to mention that the size does not change, but that only counts for the direction where "Center" is selected. It could be that in the other direction, another anchoring mode is applied.